pom.xml

```diff
@@ -87,15 +87,15 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.1.1</cdap.version>
-    <cdap.plugin.version>2.3.5</cdap.plugin.version>
+    <cdap.version>6.9.0-SNAPSHOT</cdap.version>
```

Comment:
Use 6.8.0 as cdap.version. Changes in 6.8.0 should be sufficient to support the state store functionality.

Reply:
changed the `cdap.version` to 6.8.0 2698de880d1d31e02abdfb05d2c3868a321c46c0